src/register/tests/RegistrationPage.test.jsx

```diff
@@ -938,6 +945,27 @@ describe('RegistrationPage', () => {
       expect(registrationPage.find('input#password').props().value).toEqual('password1');
       expect(registrationPage.find('.email-warning-alert-link').first().text()).toEqual('john.doe@hotmail.com');
     });
+
+    it('should set country in component state on country change with translations', () => {
```

Comment:
nit: should set the component country state on onChange while the page is translated.